meta-lmp-bsp/dynamic-layers/xilinx/recipes-bsp/u-boot/u-boot-xlnx/k26/lmp.cfg

```diff
@@ -22,7 +22,7 @@ CONFIG_TEE=y
 CONFIG_OPTEE=y
 CONFIG_OPTEE_LIB=y
 # Align max size with ATF (OCRAM - 256KB)
-CONFIG_SPL_SIZE_LIMIT=0x26000
```

Comment:
Did you boot test this change?

Reply:
Then we need to track this somewhere to avoid surprises later on, if we want to change just to get it to build again.

@angolini ^